lib/src/components/NxNexusPageHeader/NxNexusPageHeader.scss

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-container-helpers';
+@import '../../scss-shared/nx-text-helpers';
+
+.nx-page-header--nexus {
+  .nx-page-header__inner {
+    padding-left: 0;
+  }
+
+  .nx-product {
+    display: -ms-grid;
+    display: grid;
+    -ms-grid-columns: 72px 16px 1fr;
+    grid-template-columns: 72px 1fr;
+    -ms-grid-rows: 47px auto;
+    grid-template-rows: 47px auto;
```

Comment:
I'm thinking this should be 46px.  It's somewhat arbitrary, but the way I'm figuring it is by noticing that in the mockup, the top edge of the `.nx-product__details` element seems to be exactly even with the bottom-middle corners of the hexagon logo.  By that metric, the implementation currently has it just a little bit too low, and dropping this value by 1px fixes it.

Reply:
> I'm thinking this should be 46px. It's somewhat arbitrary, but the way I'm figuring it is by noticing that in the mockup, the top edge of the `.nx-product__details` element seems to be exactly even with the bottom-middle corners of the hexagon logo. By that metric, the implementation currently has it just a little bit too low, and dropping this value by 1px fixes it.

https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/598eb730547fdcb6b5452788b802f72304c19f08